python/GangaLHCb/__init__.py

```diff
@@ -85,43 +85,21 @@ def _store_root_version():
 
 
 def _store_dirac_environment():
+    from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache
     diracversion = _guess_version('LHCBDIRAC')
-    import tempfile
-    import subprocess
-    import os
     platform = os.environ['CMTOPT']
     setup_script = 'SetupProject.sh'
```

Comment:
I've tested it and it fails due to the presence of `ROOT` in the command `source lb-run LHCBDIRAC v8r2p27 ROOT`. Removing that part of the command still gives problems because the model for `lb-run` is that is not sourceable. It's the difference between:

```bash
source SetupProject.sh LHCbDirac v8r2p27; python -c "import os; print([e for e in os.environ if e.startswith(\"DIRAC\")])"
```

and
```bash
lb-run LHCBDIRAC v8r2p27 python -c "import os; print([e for e in os.environ if e.startswith(\"DIRAC\")])"
```
(Note the semi-colon and the `source` command).

The problem is that `execute()` automatically adds on the `; python -c ...` part when doing an `update_env`. It looks like using the current `update_env` will not work for `lb-run`.

I'll add another commit to this branch to make LHCb do their own, simplified version of `update_env`.